src/soter/openssl/soter_ec_key_3.c

```diff
@@ -139,10 +139,9 @@ soter_status_t soter_engine_specific_to_ec_priv_key(const soter_engine_specific_
     size_t output_length;
     char curve_str[MAX_CURVE_NAME_LEN];
     int curve;
-    // Maximum supported key length is 521 bits (secp521r1), roughly 65.12 bytes, rounding to 66,
-    // +1 byte because our private keys have unneeded zero byte at the beginning
```

Comment:
So this +1 byte is unneeded?

Reply:
actually yes, but we have legacy implementation that added one extra byte to the private key and removing it breaks backward compatibility...